searchlib/src/vespa/searchlib/attribute/singlestringpostattribute.hpp

```diff
@@ -152,5 +154,12 @@ SingleValueStringPostingAttributeT<B>::getSearch(QueryTermSimpleUP qTerm,
                                 *this);
 }
 
-} // namespace search
+template <typename B>
```

Comment:
This is so simple you can just have it in the header file.

Reply:
Fixed